### ISSUE TITLE
No button to choose quality

### ISSUE THREAD

**Issue by etiennebacher** [FreeTubeApp/FreeTube#1364]
**Behavior of the program**
There's no button to choose the quality of the video. The quality button is correctly displayed in v0.12 (AppImage) but not in v0.13 (AppImage & FlatHub). The other buttons (speed, etc.) appear normally.

**Expected behavior**
The button to choose the quality of the video should appear.

**To Reproduce**
Run any video. Notice the absence of the button.

**Screenshots**
![bug_freetube](https://user-images.githubusercontent.com/52219252/120034710-fe9fba80-bfec-11eb-9717-8b325634f85a.png)

**Environment Information (please complete the following information):**
 - OS: Kubuntu
 - OS Version: Ubuntu 18.04.5 LTS
 - FreeTube version: v0.13.0 Beta
 - Installation Method (If applicable): FlatHub & AppImage
 - Primary API Used: https://invidious.snopyta.org

**Comment by arrowgent** [comment]
yes v0.13 the quality option is missing for (some) videos

**Comment by GilgusMaximus** [comment]
I am not 100% sure what Preston changed and whether the button now accounts if only 1 option is available, but it could very well be that Invidious only provides one quality and this then cannot be changed

**Comment by etiennebacher** [comment]
I noted that this issue doesn't always appear, but when it does, it is by period. E.g:

* I run 4-5 videos in a row, none of them has the quality button
* Relaunch Freetube some time after, now the videos have this button

I will try later with other Invidious instances. Could it be related to my internet connection as well? I don't have a very good one

**Comment by GilgusMaximus** [comment]
It should not be related. If the data for the video arrived, the quality selection comes with it. 

But could you try to see whether this happens when you use the local api instead for a few videos. If it works we at least know there is an issue with Invidiosu

**Comment by etiennebacher** [comment]
Everything works well with the local API: the quality button is correctly displayed. I checked with several instances of Invidious and the issue is still there, so this is an issue with Invidious.

In my previous comment, I said that when I use Invidious, sometimes the button appears and sometimes not. I just realized that it appears when there's an Invidious error and that Freetube is falling back to the local API. 

Not sure if I should close this.
